python/nano/src/bigdl/nano/automl/tf/keras/Sequential.py

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import tensorflow as tf
+
```

Comment:
Should we reuse the `bigdl.nano.tf.keras.sequential` ?

Reply:
In order not to mess up with the current sequential I just keep it in automl for now. There're are loads of modifications in sequential and model, e.g. autogenerated decorated layers, decide whether to create a normal graph or searchable graph based on search space, lazy computation graph construction and delayed model instantiation (which makes some of the methods not usable until end_search). 

I will merge hpo and tf optimizations into tf.keras.sequential when hpo is relatively stable